scripts/pre_commit_linter.py

```diff
@@ -1580,8 +1589,7 @@ def main():
         html_directive_name_messages + import_order_messages +
         newline_messages + docstring_messages + comment_messages +
         html_tag_and_attribute_messages + html_linter_messages +
-        linter_messages + pattern_messages +
-        copyright_notice_messages)
+        linter_messages + pattern_messages + copyright_notice_messages)
```

Comment:
I don't think we need to change this...

Reply:
Isn't it better to save lines when the word limit is not violated?